tests/sles4sap/sap_deployment_automation_framework/deploy_sap_systems.pm

```diff
@@ -105,6 +105,11 @@ sub run {
     az_login();
     sdaf_execute_deployment(deployment_type => 'sap_system', timeout => 3600);
 
+    # Check if sap-parameters.yaml file was created by SDAF
+    my $sap_parameters_path = "$config_root_path/sap-parameters.yaml";
+    record_info('File check', "Check if file '$sap_parameters_path' was created by SDAF");
+    assert_script_run("test -f $sap_parameters_path");
```

Comment:
Just an alternative, not sure myself if it's more beneficial change.
You could technically merge those two checks into a loop:

```
my @check_files = (
    "$config_root_path/sap-parameters.yaml", 
    get_sdaf_inventory_path(sap_sid => $sap_sid, config_root_path => $config_root_path));

for $file (@check_files) {
    record_info('File check', "Check if file '$file' was created by SDAF");
    assert_script_run("test -f $file");
}
```